src/getEntityTypescriptPostgres.ts

```diff
@@ -307,11 +307,16 @@ async function getEntityTypescriptPostgres(
 	return { entityString, enumsStrings };
 }
 
-function addLineIfNeeded(array: string[], format: (item: string) => string) {
-	if (array.length > 0) {
-		return `\n${array.map(format).join("\n")}`;
+function addLineIfNeeded(array: string[], format?: (item: string) => string) {
+	if (array.length === 0) {
+		return ``;
 	}
-	return ``;
+
+	if (!format) {
+		return `\n${array.join("\n")}`;
+	}
+
+	return `\n${array.map(format).join("\n")}`;
```

Comment:
Not a big fan.
But if it make you happy, ok!

Reply:
yes i think it is better than passing function that just creating new array